recipes/simdjson/0.4.7/conanfile.py

```diff
@@ -48,16 +46,15 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        if self.settings.compiler.cppstd and \
-           not self.settings.compiler.cppstd in self._supported_cppstd:
-          raise ConanInvalidConfiguration("This library requires c++11 standard or higher."
-                                          " {} required."
-                                          .format(self.settings.compiler.cppstd))
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, "11")
 
-        if not self._is_supported_compiler():
-            raise ConanInvalidConfiguration("This library is tested with a family of recent compilers."
-                                            " {} {} is not supported."
-                                            .format(self.settings.compiler, self.settings.compiler.version))
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version:
+            if tools.Version(self.settings.compiler.version) < minimum_version:
+                raise ConanInvalidConfiguration("{} requires C++14, which your compiler does not fully support.".format(self.name))
```

Comment:
The message should be a little bit different for simdjson (same for the warning).

Reply:
Do you mean to change to C++11 only? Looks like simdjson is for 14.

Reply:
So why checking C++11 above? :p simdjson is for C++17 I think.

Reply:
Finally found a reference to 17 inside cmake files. Changed to 17.